samcli/commands/_utils/value_parser.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
The name of the file seems too generic `value_parser`. Can we make it more unique to this use case?